src/llm/apis/openai_completions.cpp

```diff
@@ -278,6 +276,74 @@ absl::Status OpenAIChatCompletionsHandler::parseMessages() {
     return absl::OkStatus();
 }
 
+absl::Status OpenAIChatCompletionsHandler::parseTools() {
+    auto it = doc.FindMember("tools");
+    if (it == doc.MemberEnd())
```

Comment:
what happens when someone specifies "tool_choice" but doesnt specify "tools"? As far I see we proceed with OK, meaning that user doesnt see any error, is it intended?